SignalServiceKit/src/Messages/Interactions/OWSContact.m

```diff
@@ -73,7 +62,10 @@ - (NSString *)localizedLabel
         case OWSContactPhoneType_Work:
             return [CNLabeledValue localizedStringForLabel:CNLabelWork];
         default:
-            return self.label;
+            if (self.label.ows_stripped.length < 1) {
+                return NSLocalizedString(@"CONTACT_PHONE", @"Label for a contact's phone number.");
```

Comment:
Thanks for this idea @michaelkirk

Reply:
I don't exactly remember talking about this, but I'm usually happy to get rid of a `nullable`!